tasks/gulp/backstop.js

```diff
@@ -26,7 +26,12 @@ gulp.task('backstop:approve', () => runBackstop('approve'));
 
 gulp.task('backstop-test', async () => {
   console.log(chalk.green('\nStart server for visual regression testing'));
-  const server = await appListen(port);
+
+  const server = new Promise((resolve) => {
```

Comment:
just seen this needs an await - which makes me think I haven't run backstop tests 👌